src/execution_plan/ops/op_create.c

```diff
@@ -280,20 +285,11 @@ OpResult OpCreateConsume(OpBase *opBase, Record *r) {
     }
 
     if(op->op.childCount == 0) return OP_DEPLETED;
-    else return res;
+    return res;
 }
 
 OpResult OpCreateReset(OpBase *ctx) {
```

Comment:
Is it safe for this reset to be a no-op?

Reply:
I think so, currently only cartesian product issue a restart, which propagates upwards and so reset will never reach OpCreate

Reply:
:+1: